@vates/parse-duration/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "newLine": "lf",
+    "noImplicitAny": true,
+    "strict": false,
+    "skipLibCheck": true,
+    "baseUrl": ".",
+    "sourceMap": false,
+    "strictNullChecks": true,
+    "target": "ES2015",
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "exclude": ["node_modules", "dist"]
+}
```

Comment:
I don't like this config, too many settings, and too many I don't understand.

What do you think of delegating the tedious configuration to the community?

Something like: https://github.com/julien-f/untitled/blob/dba7b3404da054f906a0f017daede66f7378e281/tsconfig.json

Reply:
We need to keep control over this file. More than just a configuration, it is what allows us to define the quality of code we want.